backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/CommitOfManagedBranch.java

```diff
@@ -20,11 +20,17 @@ public class CommitOfManagedBranch implements ICommitOfManagedBranch {
   private final IGitCoreCommit coreCommit;
 
   @Override
-  @ToString.Include(name = "message")
+  @ToString.Include(name = "shortMessage")
   public String getShortMessage() {
     return coreCommit.getShortMessage();
   }
 
+  @Override
+  @ToString.Include(name = "fullMessage")
```

Comment:
Actually... this `toStrings` methods are only really ever needed for the sake of debug & logs... TBH I don't think we'll even need `fullMessage` there (seems unnecessarily verbose). I'd just `@ToString.Include` only `shortMessage`